Dockerfile.dapper

```diff
@@ -55,6 +57,16 @@ RUN cd /usr/src && \
     make && \
     make install
 
+# Install libqcow to resolve error:
+#   vendor/github.com/longhorn/longhorn-engine/pkg/qcow/libqcow.go:6:11: fatal error: libqcow.h: No such file or directory
```

Comment:
Can we just use installing libqcow-devel package?

I am also confused why the previous builds work?

Reply:
I am not sure. I'm using the same install as the engine
https://github.com/longhorn/longhorn-engine/blob/045265cd71d1d3c01ead215da49c1a0c1b40ebaf/Dockerfile.dapper#L51-L58

Maybe we can have another issue to deal with this.

Reply:
Yeah, let's tackle another issue, but it seems right now longhorn-instance-manager depends on longhorn-engine proto.

Reply:
https://github.com/longhorn/longhorn/issues/3976